src/style/app.module.css

```diff
@@ -1942,185 +2058,794 @@ input[type='radio']:checked + label:hover {
+  outline: none;
+  border: none;
+  color: #000;
+  font-weight: 900;
+}
+
+.startPostBtn:hover {
+  background-color: #00000010;
+  border: 0;
+  color: #000 !important;
+}
+.icons {
+  width: 25px;
+}
+
+.icons svg {
+  stroke: #000;
+}
+
+.icons.dark {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background-color: transparent;
+}
+
+.icons.dark svg {
+  stroke: #000;
+}
+
+.iconLabel {
+  margin: 0;
+  color: #000;
+  font-weight: 900;
+}
+
+.uploadLink {
+  text-align: center;
+  width: min-content;
+  padding: 8px 4px;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.uploadLink:hover {
+  background-color: #00000010;
+}
+
+.modal {
+  width: 100dvw;
+  margin: 0 auto;
+}
+
+.imageInput {
+  display: none;
+}
+
+.previewImage {
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 1rem;
+}
+
+.previewImage img {
+  border-radius: 8px;
+}
+
+.imageContainerPromotedPost {
+  max-width: 100%;
+}
+
+.cardHeaderPromotedPost {
   display: flex;
+  flex-direction: row;
+  gap: 10px;
+  align-items: center;
+  color: var(--light-blue);
 }
 
-.dropdownModalToggle {
-  width: 50%;
+.borderBox {
+  border: 1px solid #dddddd;
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
+  border-radius: 4px;
 }
 
-.greenregbtn {
-  margin-top: 1rem;
-  border: 1px solid var(--bs-gray-300);
-  box-shadow: 0 2px 2px var(--bs-gray-300);
-  padding: 10px 10px;
+.borderRounded5 {
   border-radius: 5px;
-  background-color: var(--bs-primary);
-  width: 100%;
-  font-size: 16px;
-  color: var(--bs-white);
-  outline: none;
-  font-weight: 600;
-  cursor: pointer;
-  transition:
-    transform 0.2s,
-    box-shadow 0.2s;
-  width: 100%;
 }
 
-.manageBtn {
-  margin: 1rem 0 0;
-  margin-top: 15px;
-  border: 1px solid var(--grey-border-box-color);
-  box-shadow: 0 2px 2px var(--grey-border-box-color);
-  padding: 10px 10px;
-  border-radius: 5px;
-  font-size: 16px;
-  color: var(--bs-white);
-  outline: none;
-  font-weight: 600;
-  cursor: pointer;
-  width: 45%;
-  transition:
-    transform 0.2s,
-    box-shadow 0.2s;
+.borderRounded8 {
+  border-radius: 8px;
 }
 
-.removeFilterIcon {
-  cursor: pointer;
+.topRadius {
+  border-top-left-radius: 24px;
+  border-top-right-radius: 24px;
 }
 
-.searchForm {
-  display: inline;
+.bottomRadius {
+  border-bottom-left-radius: 24px;
+  border-bottom-right-radius: 24px;
 }
 
-.view {
-  margin-left: 2%;
-  font-weight: 600;
-  font-size: 16px;
-  color: var(--bs-gray-600);
+.shadow {
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
 }
 
-/* header (search, filter, dropdown) */
-.btncon .btnsContainer {
-  display: flex;
-  margin: 0.5rem 0 1.5rem 0;
+.mainContainerUserPeople {
+  width: 50%;
+  flex-grow: 3;
 }
 
-.btncon .btnsContainer .input {
-  flex: 1;
-  min-width: 18rem;
-  position: relative;
+.semiBold {
+  font-weight: 500;
 }
 
-.btncon .btnsContainer input {
-  outline: 1px solid var(--bs-gray-400);
+.placeholderColor::placeholder {
+  color: #737373;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`